packages/text-editor/src/utils.ts

```diff
@@ -51,26 +45,46 @@ export function getDataAttribute (
   }
 }
 
-function getAvatarColor (name: string, avatar: string, darkTheme: boolean): ColorDefinition {
-  const [type, color] = avatar.split('://')
-  if (type === AvatarType.COLOR) {
-    return getPlatformAvatarColorByName(color, darkTheme)
-  }
-  return getPlatformAvatarColorForTextDef(name, darkTheme)
-}
-
 export async function getCollaborationUser (): Promise<CollaborationUser> {
   const client = getClient()
 
   const me = getCurrentAccount() as PersonAccount
   const person = await client.findOne(contact.class.Person, { _id: me.person })
   const name = person !== undefined ? formatName(person.name) : me.email
-  const color = getAvatarColor(name, person?.avatar ?? '', get(themeStore).dark)
+  const color = getColorNumberByText(name)
 
   return {
     id: me._id,
     name,
     email: me.email,
-    color: color.icon ?? 'var(--theme-button-default)'
+    color
+  }
+}
+
+export function throttle (fn: (...args: any[]) => any, delay: number): any {
+  let timeout: any | undefined
+
+  return (...args: any[]) => {
+    if (timeout === undefined) {
+      fn(...args)
+
+      timeout = setTimeout(() => {
+        timeout = undefined
+      }, delay)
+    }
+  }
+}
+
+export function debounce (fn: (...args: any[]) => any, delay: number): any {
```

Comment:
Please use  `DelayedCaller` from ui package.

Reply:
I will unify this and move to core or ui. But I don't like `DelayedCaller` name, because usually `throttle` and `debounce` are used.